[Release] Release v0.228.0 (#1752)
databricks/cli: CHANGELOG.md

# Version changelog

## [Release] Release v0.228.0

CLI:
 * Do not error if we cannot prompt for a profile in `auth login` ([#1745](https://github.com/databricks/cli/pull/1745)).

Bundles:

> As of this release CLI will show a prompt is if there are configuration changes which will lead to a DLT recreation.
> Users can skip the prompt by specifying the `--auto-approve` flag

Reviewer: ```suggestion
As of this release, the CLI will show a prompt if there are configuration changes that lead to DLT pipeline recreation.
Users can skip the prompt by specifying the `--auto-approve` flag.
```
